e2e-tests/package.json

```diff
@@ -7,8 +7,7 @@
 		"fmt-check": "prettier ./tests --check",
 		"fmt": "prettier ./tests --write",
 		"build": "cp -r ../pallets/laos-evolution/src/precompiles/evolution_collection_factory/contracts contracts && cp -r ../pallets/laos-evolution/src/precompiles/evolution_collection/contracts contracts && cp -r ../pallets/asset-metadata-extender/src/precompiles/asset_metadata_extender/contracts contracts && truffle compile && rm -rf contracts",
-		"test": "mocha -r ts-node/register -t 70000 --grep '@qa' --invert 'tests/**/*.ts'",
-		"qa": "mocha -r ts-node/register -t 200000 --grep '@qa' 'tests/**/*.ts'",
+		"test": "mocha -r ts-node/register 'tests/**/*.ts'",
 		"test-sql": "FRONTIER_BACKEND_TYPE='sql' mocha -r ts-node/register 'tests/**/*.ts'"
```

Comment:
we might keep timeout param otherwise test will timeout

Reply:
addressed. the timeout is now the sum of the previous two (qa and non-qa)